s3/src/main/scala/akka/stream/alpakka/s3/impl/S3Stream.scala

```diff
@@ -78,9 +84,12 @@ private[alpakka] final class S3Stream(credentials: AWSCredentials, region: Strin
     import mat.executionContext
 
     val req = HttpRequests.initiateMultipartUploadRequest(s3Location, contentType, cannedAcl)
+
     val response = for {
       signedReq <- Signer.signedRequest(req, signingKey)
+      _ <- Future(system.log.info("Request: {}", signedReq))
```

Comment:
instead of using `system.log` you should use a more specific `LoggingAdapter via

```scala
val log = Logging(system, getClass)
```